src/main/scala/no/ndla/imageapi/controller/RawController.scala

```diff
@@ -38,7 +39,8 @@ trait RawController {
       queryParam[Option[Int]]("cropEndY").description("The end image coordinate Y, in percent (0 to 100), specifying the crop end position. If used the other crop parameters must also be supplied"),
       queryParam[Option[Int]]("focalX").description("The end image coordinate X, in percent (0 to 100), specifying the focal point. If used the other focal point parameter, width and/or height, must also be supplied"),
       queryParam[Option[Int]]("focalY").description("The end image coordinate Y, in percent (0 to 100), specifying the focal point. If used the other focal point parameter, width and/or height, must also be supplied"),
-      queryParam[Option[Double]]("ratio").description("The wanted aspect ratio, defined as width/height. To be used together with the focal parameters. If used the width and height is ignored and derived from the aspect ratio instead.")
+      queryParam[Option[Double]]("ratio").description("The wanted aspect ratio, defined as width/height. To be used together with the focal parameters. If used the width and height is ignored and derived from the aspect ratio instead."),
+      queryParam[Option[Double]]("storedRatio").description("Use stored crop and focal point parameters for given aspect ratio, or use defaults instead.")
```

Comment:
Hadde også for query-parametre foretrukket `-`, men ser at vi benytter camelCase fra før her..

Reply:
Ja, ser den. Gjør ingenting med dette nå, ellers brekker vi APIet.